pkgs/development/libraries/smpeg2/default.nix

```diff
@@ -1,21 +1,28 @@
-{ stdenv, darwin, fetchsvn, autoconf, automake, pkgconfig, makeWrapper, SDL2 }:
+{ stdenv
+, autoconf
+, automake
+, darwin
+, fetchsvn
+, makeWrapper
+, pkg-config
+, SDL2
+}:
 
 stdenv.mkDerivation rec {
-  name = "smpeg2-svn${version}";
-  version = "412";
+  pname = "smpeg2";
+  version = "unstable-2017-10-18";
 
   src = fetchsvn {
     url = "svn://svn.icculus.org/smpeg/trunk";
-    rev = version;
-    sha256 = "1irf2d8f150j8cx8lbb0pz1rijap536crsz0mw871xrh6wd2fd96";
+    rev = "413";
+    sha256 = "193amdwgxkb1zp7pgr72fvrdhcg3ly72qpixfxxm85rzz8g2kr77";
   };
 
   patches = [
-    ./gcc6.patch
```

Comment:
please also delete this one.

Reply:
I am not sure what you mean by this as you are commenting on a deleted line and the file `gcc6.patch` has been renamed to `hufftable-uint_max.patch`. Are you suggesting to drop the patch completely?

Reply:
Just to be sure, I tried to remove the `hufftable-uint_max.patch` and the build fails with

```
audio/hufftable.cpp:587:1: error: narrowing conversion of '-1' from 'int' to 'unsigned int' [-Wnarrowing]
```

Reply:
Right. I did not spot the rename.